docs/How-to-use-kaito-in-aks-arc.md

```diff
@@ -0,0 +1,137 @@
+# Use Kaito in AKS Arc
+This article describes how to deploy AI models on AKS arc with AI toolchain operator (KAITO). The AI toolchain operator (KAITO) is a managed add-on for all AKS, and it simplifies the experience of running OSS AI models on your AKS clusters. You may follow the workflow below to enable this feature:

```

Comment:
Since this doc is in Kaito repo, you may remove the 2nd sentence that explains Kaito. Instead, you'd better briefly explain ARC in the introduction.